app/transformers/rural-payments/business.js

```diff
@@ -89,3 +89,27 @@ export const transformOrganisationToBusiness = (data) => {
     sbi: `${data?.sbi}`
   }
 }
+
+export function transformCountyParishHoldings(data) {
+  return [...data]
+    .sort((a, b) => {
+      if (a.cph_number < b.cph_number) {
+        return -1
+      }
+
+      if (a.cph_number > b.cph_number) {
+        return 1
+      }
+
+      return new Date(b.start_date) - new Date(a.start_date)
+    })
```

Comment:
This could be a bit simpler :shrug: 
```suggestion
    .sort((a, b) => a.cph_number - b.cph_number || new Date(b.start_date) - new Date(a.start_date))
```

Reply:
Thanks. The subtraction didn't work because `cps_numbers` are in fact strings: `CC/PPP/HHHH`

- CC = County code
- PPP = Parish code
- HHHH = Holding number (identifies the specific farm or premises)

Have slimmed it down with `localeCompare` though; that seems to do the trick.

Reply:
Have updated to sort numerically; first by county, then parish, then holding number. And then by the start date.

Reply:
Excellent!